### ISSUE TITLE
Opening TIC-80 App (Latest Dev Version) result crash on Android Version 11

### ISSUE THREAD

**Issue by MasterNama17083** [nesbox/TIC-80#2134]
I installed the latest version and then opening the app will get crash, because probably didn't work?
https://user-images.githubusercontent.com/80534869/219957018-314e5463-9d3d-44ef-af09-865f3e8db951.mp4

I'm on Android version 11, and Mi A3 (device name).

**Comment by sogaiu** [comment]
I also get a crash -- I think there may be some info available via `adb logcat`.

**Comment by MasterNama17083** [comment]
> I also get a crash, though for Android 13 -- I think there may be some info available via `adb logcat`.

I'll try to get adb logcat using [SDK Platform Tools](https://developer.android.com/studio/releases/platform-tools) to get a dump to debug log and let's see what we got a problem here:
_(I spent an hour doing some time wasting lol, sorry.)_
```
--------- beginning of crash
02-19 21:31:40.854 12203 12203 F DEBUG   : pid: 12174, tid: 12174, name: SDLActivity  >>> com.nesbox.tic <<<
02-19 21:31:40.854 12203 12203 F DEBUG   : uid: 10264
02-19 21:31:40.854 12203 12203 F DEBUG   : signal 6 (SIGABRT), code -1 (SI_QUEUE), fault addr --------
02-19 21:31:40.854 12203 12203 F DEBUG   : Abort message: 'No pending exception expected: java.lang.NoSuchMethodError: no static or non-static method "Lorg/libsdl/app/SDLActivity;.nativeGetVersion()Ljava/lang/String;"
02-19 21:31:40.854 12203 12203 F DEBUG   :   at java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader, java.lang.Class) (Runtime.java:-2)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at java.lang.String java.lang.Runtime.nativeLoad(java.lang.String, java.lang.ClassLoader) (Runtime.java:1131)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void java.lang.Runtime.loadLibrary0(java.lang.ClassLoader, java.lang.Class, java.lang.String) (Runtime.java:1085)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void java.lang.Runtime.loadLibrary0(java.lang.Class, java.lang.String) (Runtime.java:1008)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void java.lang.System.loadLibrary(java.lang.String) (System.java:1664)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void org.libsdl.app.SDL.loadLibrary(java.lang.String) (SDL.java:73)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void org.libsdl.app.SDLActivity.loadLibraries() (SDLActivity.java:179)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void org.libsdl.app.SDLActivity.onCreate(android.os.Bundle) (SDLActivity.java:227)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.Activity.performCreate(android.os.Bundle, android.os.PersistableBundle) (Activity.java:8000)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.Activity.performCreate(android.os.Bundle) (Activity.java:7984)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.Instrumentation.callActivityOnCreate(android.app.Activity, android.os.Bundle) (Instrumentation.java:1310)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at android.app.Activity android.app.ActivityThread.performLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.content.Intent) (ActivityThread.java:3403)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at android.app.Activity android.app.ActivityThread.handleLaunchActivity(android.app.ActivityThread$ActivityClientRecord, android.app.servertransaction.PendingTransactionActions, android.content.Intent) (ActivityThread.java:3594)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.servertransaction.LaunchActivityItem.execute(android.app.ClientTransactionHandler, android.os.IBinder, android.app.servertransaction.PendingTransactionActions) (LaunchActivityItem.java:85)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.servertransaction.TransactionExecutor.executeCallbacks(android.app.servertransaction.ClientTransaction) (TransactionExecutor.java:135)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.servertransaction.TransactionExecutor.execute(android.app.servertransaction.ClientTransaction) (TransactionExecutor.java:95)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:2067)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:106)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.os.Looper.loop() (Looper.java:223)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:7698)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at java.lang.Object java.lang.reflect.Method.invoke(java.lang.Object, java.lang.Object[]) (Method.java:-2)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run() (RuntimeInit.java:592)
02-19 21:31:40.854 12203 12203 F DEBUG   :   at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:952)
02-19 21:31:40.854 12203 12203 F DEBUG   : '
02-19 21:31:40.854 12203 12203 F DEBUG   :     x0  0000000000000000  x1  0000000000002f8e  x2  0000000000000006  x3  0000007febf54380
02-19 21:31:40.854 12203 12203 F DEBUG   :     x4  0000007901fdc000  x5  0000007901fdc000  x6  0000007901fdc000  x7  000000000001ddec
02-19 21:31:40.854 12203 12203 F DEBUG   :     x8  00000000000000f0  x9  759862a5d643b707  x10 0000000000000000  x11 ffffffc0fffffbdf
02-19 21:31:40.854 12203 12203 F DEBUG   :     x12 0000000000000001  x13 0000000000000c7e  x14 00000078fdfc9d22  x15 0000000000000005
02-19 21:31:40.854 12203 12203 F DEBUG   :     x16 00000078fdfc6c80  x17 00000078fdfa8b70  x18 0000007901706000  x19 0000000000002f8e
02-19 21:31:40.854 12203 12203 F DEBUG   :     x20 0000000000002f8e  x21 00000000ffffffff  x22 0000000000000058  x23 0000000000000058
02-19 21:31:40.854 12203 12203 F DEBUG   :     x24 000000760b5f7000  x25 0000000000000043  x26 000000790053886c  x27 000000760b5f9000
02-19 21:31:40.854 12203 12203 F DEBUG   :     x28 000000760b5fa000  x29 0000007febf54400
02-19 21:31:40.854 12203 12203 F DEBUG   :     lr  00000078fdf5c3e0  sp  0000007febf54360  pc  00000078fdf5c40c  pst 0000000000000000
02-19 21:31:41.122 12203 12203 F DEBUG   : backtrace:
02-19 21:31:41.122 12203 12203 F DEBUG   :       #00 pc 000000000004e40c  /apex/com.android.runtime/lib64/bionic/libc.so (abort+164) (BuildId: 8d77279a411c99f8bc6edb79c76340fb)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #01 pc 000000000055d080  /apex/com.android.art/lib64/libart.so (art::Runtime::Abort(char const*)+2308) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #02 pc 0000000000013978  /system/lib64/libbase.so (android::base::SetAborter(std::__1::function<void (char const*)>&&)::$_3::__invoke(char const*)+76) (BuildId: 7f81cc97d60164bfae3b52a2cd822f20)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #03 pc 0000000000012fa4  /system/lib64/libbase.so (android::base::LogMessage::~LogMessage()+320) (BuildId: 7f81cc97d60164bfae3b52a2cd822f20)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #04 pc 00000000005b1b84  /apex/com.android.art/lib64/libart.so (art::Thread::AssertNoPendingException() const+1808) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #05 pc 00000000001d1950  /apex/com.android.art/lib64/libart.so (art::ClassLinker::FindClass(art::Thread*, char const*, art::Handle<art::mirror::ClassLoader>)+64) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #06 pc 0000000000393080  /apex/com.android.art/lib64/libart.so (art::JNI<false>::FindClass(_JNIEnv*, char const*)+1048) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #07 pc 0000000000130340  /data/app/~~YMkic6rvJzozuSaom_x45A==/com.nesbox.tic-t-TcF21wVRsTYTvcC3GxSA==/lib/arm64/libtic80.so (BuildId: 35cca73b4ee5a5c314e8a444555146f22ec750b0)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #08 pc 00000000001302b0  /data/app/~~YMkic6rvJzozuSaom_x45A==/com.nesbox.tic-t-TcF21wVRsTYTvcC3GxSA==/lib/arm64/libtic80.so (JNI_OnLoad+148) (BuildId: 35cca73b4ee5a5c314e8a444555146f22ec750b0)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #09 pc 0000000000389178  /apex/com.android.art/lib64/libart.so (art::JavaVMExt::LoadNativeLibrary(_JNIEnv*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, _jobject*, _jclass*, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >*)+3348) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #10 pc 000000000000516c  /apex/com.android.art/lib64/libopenjdkjvm.so (JVM_NativeLoad+416) (BuildId: 7f33505c1a996bcb1103edc6fb3a022e)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #11 pc 0000000000080a24  /apex/com.android.art/javalib/arm64/boot.oat (art_jni_trampoline+228) (BuildId: 96a2ad2581099975b8c7bb4ab40fb39a7086a166)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #12 pc 00000000000996e8  /apex/com.android.art/javalib/arm64/boot.oat (java.lang.Runtime.loadLibrary0+328) (BuildId: 96a2ad2581099975b8c7bb4ab40fb39a7086a166)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #13 pc 000000000009aaf4  /apex/com.android.art/javalib/arm64/boot.oat (java.lang.Runtime.loadLibrary0+180) (BuildId: 96a2ad2581099975b8c7bb4ab40fb39a7086a166)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #14 pc 000000000009e3a0  /apex/com.android.art/javalib/arm64/boot.oat (java.lang.System.loadLibrary+96) (BuildId: 96a2ad2581099975b8c7bb4ab40fb39a7086a166)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #15 pc 00000000001337e8  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #16 pc 00000000001a8a94  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+228) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #17 pc 000000000031831c  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+376) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #18 pc 000000000030e648  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+996) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #19 pc 000000000067f49c  /apex/com.android.art/lib64/libart.so (MterpInvokeStatic+548) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #20 pc 000000000012d994  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #21 pc 000000000000d2e0  /data/app/~~YMkic6rvJzozuSaom_x45A==/com.nesbox.tic-t-TcF21wVRsTYTvcC3GxSA==/oat/arm64/base.vdex (org.libsdl.app.SDL.loadLibrary+212)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #22 pc 0000000000305c44  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.11487796752256266877)+268) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #23 pc 000000000030dc34  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToInterpreterBridge(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame*, art::JValue*)+200) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #24 pc 000000000030e62c  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+968) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #25 pc 000000000067f49c  /apex/com.android.art/lib64/libart.so (MterpInvokeStatic+548) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #26 pc 000000000012d994  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_static+20) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #27 pc 000000000000a04c  /data/app/~~YMkic6rvJzozuSaom_x45A==/com.nesbox.tic-t-TcF21wVRsTYTvcC3GxSA==/oat/arm64/base.vdex (org.libsdl.app.SDLActivity.loadLibraries+20)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #28 pc 0000000000683228  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualQuick+1344) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #29 pc 0000000000131594  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_quick+20) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #30 pc 000000000000a40a  /data/app/~~YMkic6rvJzozuSaom_x45A==/com.nesbox.tic-t-TcF21wVRsTYTvcC3GxSA==/oat/arm64/base.vdex (org.libsdl.app.SDLActivity.onCreate+198)
02-19 21:31:41.122 12203 12203 F DEBUG   :       #31 pc 0000000000305c44  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.11487796752256266877)+268) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #32 pc 000000000066b24c  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+780) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #33 pc 000000000013cff8  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #34 pc 00000000005ba1ac  /system/framework/arm64/boot-framework.oat (android.app.Activity.performCreate+492) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #35 pc 0000000000320e20  /system/framework/arm64/boot-framework.oat (android.app.Instrumentation.callActivityOnCreate+80) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #36 pc 00000000003fe754  /system/framework/arm64/boot-framework.oat (android.app.ActivityThread.performLaunchActivity+2356) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #37 pc 0000000000404d1c  /system/framework/arm64/boot-framework.oat (android.app.ActivityThread.handleLaunchActivity+556) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #38 pc 00000000005c6ff4  /system/framework/arm64/boot-framework.oat (android.app.servertransaction.LaunchActivityItem.execute+404) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #39 pc 000000000034d500  /system/framework/arm64/boot-framework.oat (android.app.servertransaction.TransactionExecutor.executeCallbacks+560) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #40 pc 000000000034d250  /system/framework/arm64/boot-framework.oat (android.app.servertransaction.TransactionExecutor.execute+752) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #41 pc 00000000003ea098  /system/framework/arm64/boot-framework.oat (android.app.ActivityThread$H.handleMessage+536) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #42 pc 0000000000637334  /system/framework/arm64/boot-framework.oat (android.os.Handler.dispatchMessage+180) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #43 pc 000000000063a82c  /system/framework/arm64/boot-framework.oat (android.os.Looper.loop+1516) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #44 pc 00000000003fdb40  /system/framework/arm64/boot-framework.oat (android.app.ActivityThread.main+752) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #45 pc 00000000001337e8  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #46 pc 00000000001a8a94  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+228) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #47 pc 0000000000555748  /apex/com.android.art/lib64/libart.so (art::InvokeMethod(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jobject*, _jobject*, unsigned long)+1364) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #48 pc 00000000004d4ef0  /apex/com.android.art/lib64/libart.so (art::Method_invoke(_JNIEnv*, _jobject*, _jobject*, _jobjectArray*)+52) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #49 pc 00000000000896f4  /apex/com.android.art/javalib/arm64/boot.oat (art_jni_trampoline+180) (BuildId: 96a2ad2581099975b8c7bb4ab40fb39a7086a166)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #50 pc 000000000088f328  /system/framework/arm64/boot-framework.oat (com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run+136) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #51 pc 0000000000897a58  /system/framework/arm64/boot-framework.oat (com.android.internal.os.ZygoteInit.main+2280) (BuildId: fd4031684b0826e214a4d92dfbb116e05a8b0238)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #52 pc 00000000001337e8  /apex/com.android.art/lib64/libart.so (art_quick_invoke_static_stub+568) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #53 pc 00000000001a8a94  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+228) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #54 pc 0000000000554184  /apex/com.android.art/lib64/libart.so (art::JValue art::InvokeWithVarArgs<art::ArtMethod*>(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, art::ArtMethod*, std::__va_list)+448) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #55 pc 0000000000554638  /apex/com.android.art/lib64/libart.so (art::JValue art::InvokeWithVarArgs<_jmethodID*>(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, _jmethodID*, std::__va_list)+92) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #56 pc 0000000000438b08  /apex/com.android.art/lib64/libart.so (art::JNI<true>::CallStaticVoidMethodV(_JNIEnv*, _jclass*, _jmethodID*, std::__va_list)+656) (BuildId: c58eb791dd4db5a5fc1ef4a3404cd082)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #57 pc 000000000009a424  /system/lib64/libandroid_runtime.so (_JNIEnv::CallStaticVoidMethod(_jclass*, _jmethodID*, ...)+124) (BuildId: 8e23540368b2f67254e475699a5032b0)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #58 pc 00000000000a2260  /system/lib64/libandroid_runtime.so (android::AndroidRuntime::start(char const*, android::Vector<android::String8> const&, bool)+836) (BuildId: 8e23540368b2f67254e475699a5032b0)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #59 pc 0000000000003674  /system/bin/app_process64 (main+1580) (BuildId: f0690d7ea4979aa08ecaaff244e04980)
02-19 21:31:41.123 12203 12203 F DEBUG   :       #60 pc 00000000000499e4  /apex/com.android.runtime/lib64/bionic/libc.so (__libc_init+108) (BuildId: 8d77279a411c99f8bc6edb79c76340fb)
02-19 21:31:41.321  1521 12207 I DropBoxManagerService: add tag=data_app_native_crash isTagEnabled=true flags=0x2
02-19 21:31:41.321  1521 12206 W ActivityTaskManager:   Force finishing activity com.nesbox.tic/.TIC
```

**Comment by nesbox** [comment]
Crashed on my side too.

**Comment by nesbox** [comment]
fixed here 8d2ab4353b41a3df16996b50a0cb3696958fe740
